src/matrix/registration/stages/TokenAuth.ts

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2022 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import {AuthenticationData} from "../types";
+import {BaseRegistrationStage} from "./BaseRegistrationStage";
+
+export const TOKEN_AUTH_TYPE = "org.matrix.msc3231.login.registration_token";
```

Comment:
So MSC3231 has been merged and the PR to [remove the unstable prefix](https://github.com/matrix-org/synapse/issues/11949) is also merged but a release containing this change has not been made. So this is okay for now?